traffic_ops/app/conf/cdn.conf

```diff
@@ -43,6 +43,7 @@
     },
     "portal" : {
         "base_url" : "http://localhost:8080/!#/",
+        "external_url" : "https://traffic-control-cdn.readthedocs.io/en/latest/overview/traffic_portal.html",
```

Comment:
i know i'm nitpicking here but it's not really an "external url" but more of a "docs url", right?

Reply:
by default it will be the docs URL, but we'll override the default with the actual TP URL that we want people to use instead of the old mojolicious UI

Reply:
actually, the "base_url" is the url of TP

Reply:
The `base_url` is the local URL, e.g. `https://localhost`, this needs to be the external url, e.g. `https://cdn.example.net`.

Reply:
base_url is the url where TP is running. by default, it would typically be running on localhost in a dev enviro, but this will always be changed in a prod enviro to something like tp.cdn.com/!#/